README.md

```diff
@@ -11,6 +11,14 @@ high-level languages, providing convenient wrappers around C functions. These
 bindings are intended to be used by Ethereum clients, to avoid re-implementation
 of crucial cryptographic functions.
 
+For all bindings, you first need to make sure that the `blst` submodule is correctly pulled after clone.
+To do so, you may need to run:
+
+```
+git submodule update --init
```

Comment:
Hmm we should try to fix the Makefiles so this isn't necessary. We can do something like this:

```
$(MAKE) -C ../../src blst
```